scene/gui/text_edit.cpp

```diff
@@ -5843,6 +5843,21 @@ bool TextEdit::is_line_comment(int p_line) const {
 	return false;
 }
 
+int TextEdit::get_last_line_with_content() const {
+
+	const int last_index = get_line_count() - 1;
+	const int line_timeout = get_line_count() > 50 ? 50 : get_line_count() - 1;
```

Comment:
It might be useful to add a comment as to why there's a timeout. I suppose it's done to avoid an infinite loop?

Reply:
Yeah, it's basically a sanity test in case someone has a needless amount of whitespace at the end of their document. 

Realistically, we could probably get rid of it and be safe most of the time, but I think it might be better to be safe and not majorly waste time on unnecessary looping.

I'll go ahead and add a comment to explain this though.

Reply:
Sorry for the delay @Calinou, I've been a bit preoccupied this month. I'll go ahead and add those comments later this week.